client/question/services/EventHandlerService.js

```diff
@@ -27,6 +27,7 @@ tie.factory('EventHandlerService', [
     var _currentSessionId = null;
     var _currentQuestionId = null;
     var _currentQuestionVersion = null;
+    var MAX_NUM_CODE_SUBMIT_EVENTS_PER_BATCH = 4;
```

Comment:
We might want to declare this constant next to where the batch timeframe is declared, since those two constants tend to be related to each other.

Reply:
fine with that

Reply:
done!